src/main/java/org/kainos/ea/api/EmployeeService.java

```diff
@@ -1,17 +1,46 @@
 package org.kainos.ea.api;
 
 import org.kainos.ea.cli.Employee;
+import org.kainos.ea.cli.EmployeeRequest;
+import org.kainos.ea.client.EmployeeDoesNotExistException;
+import org.kainos.ea.client.FailedToUpdateEmployeeException;
+import org.kainos.ea.client.InvalidEmployeeException;
+import org.kainos.ea.core.EmployeeValidator;
 import org.kainos.ea.db.EmployeeDao;
 
 import java.sql.SQLException;
 import java.util.List;
 
 public class EmployeeService {
 
+    private EmployeeValidator employeeValidator = new EmployeeValidator();
     private EmployeeDao employeeDao = new EmployeeDao();
 
     public List<Employee> getAllEmployee() throws SQLException {
 
         return employeeDao.getAllEmployees();
     }
+
+public void updateEmployee(int id, EmployeeRequest employee) throws InvalidEmployeeException, EmployeeDoesNotExistException, FailedToUpdateEmployeeException {
+
+        try{
+            boolean validation = employeeValidator.isValidEmployee(id);
```

Comment:
Could you potentially merge `employeeValidator.isValidEmployee` and `employeeDao.getEmployeeByID` into 1 method call?